t/t-checkout.sh

```diff
@@ -51,6 +51,14 @@ begin_test "checkout"
   grep 'accepting "file1.dat"' checkout.log
   grep 'rejecting "file1.dat"' checkout.log && exit 1
 
+  git rm file1.dat
+
+  echo "checkout should skip replacing files deleted in index"
+  GIT_TRACE=1 git lfs checkout 2>&1 | tee checkout.log
```

Comment:
I think this might be simplified to just `git lfs checkout`, since the output of the command is not subsequently examined (and so neither is any tracing out).  That would be similar to what's done immediately below in the `checkout with filters` check, where `git lfs checkout file2.dat` is run, also without examining its output.

Reply:
Good point.  I'll squash that change in.